VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/judge-hearing-list/judge-hearing-list.component.ts

```diff
@@ -52,6 +54,7 @@ export class JudgeHearingListComponent implements OnInit, OnDestroy {
         });
         this.retrieveHearingsForUser();
         this.setupSubscribers();
+        this.hearingVenueFlagsService.setHearingVenueIsScottish(false);
```

Comment:
Just thinking is there other places a judge could navigate to that should have the default crest? Having it set here might not be good enough. The service might need to subscribe to navigation and check if the route isn't linked to a conference and reset it that way. Or maybe we could look at canDeactivate guards but that doesn't seem like the intent of them but maybe there is something similar.

Reply:
once the judge selects a hearing that is in a scottish venue, we will then render the scottish crest, when they go back to the hearing they will see the hmcts crest again. This is what is in the A/C, anything else we are trying to cover is going to fall out of scope of this ticket, but yea we might want to raise other tickets to address those